experimental/dds/tree2/src/test/feature-libraries/modular-schema/modularChangeFamily.spec.ts

```diff
@@ -664,13 +716,25 @@ describe("ModularChangeFamily", () => {
 		});
 	});
 
+	const jsonValidatorFail = {
+		compile: <Schema extends TSchema>(schema: Schema) => {
+			return {
+				check: (data: unknown): data is Static<Schema> => false,
+			};
+		},
+	};
+	const dummyFamilyForExcetions = new ModularChangeFamily(fieldKinds, {
+		jsonValidator: jsonValidatorFail,
+	});
 	describe("Encoding", () => {
 		const encodingTestData: [string, ModularChangeset][] = [
 			["without constrain", rootChange1a],
-			["with constrain", rootChange3],
+			["with value constrain", rootChange3],
```

Comment:
the existing test cases look like they're missing a `t`... all of these should be "constraint" rather than "constrain"